src/js/models/v2/awardsV2/BaseIdv.js

```diff
@@ -31,6 +33,12 @@ BaseIdv.populate = function populate(data) {
     this.parentAward = data.parent_award_piid || '';
     this.parentId = data.parent_generated_unique_award_id || '';
 
+    if (data.parent_award) {
+        const parentAwards = Object.create(BaseParentAwardDetails);
```

Comment:
Making this property plural is confusing, since an award should only have one parent. I would call it `parentAward` or `parentAwardDetails`.